src/Cli/dotnet/Commands/Project/Convert/ProjectConvertCommand.cs

```diff
@@ -70,7 +73,14 @@ public override int Execute()
 
             string targetItemDirectory = Path.GetDirectoryName(targetItemFullPath)!;
             Directory.CreateDirectory(targetItemDirectory);
-            File.Copy(item.FullPath, targetItemFullPath);
+            if (keepSourceFiles)
```

Comment:
"Source files" seems a bit ambiguous to me. When I see that term I might assume `.json`, `.resx`, and so on are not included, and only source code files like `.cs` are included. I am unsure how to improve that, though.